CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
     error.
     ([#109](https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/109))
 
+-   Add `OlmMachine.dehydratedDevices()` and `DehydratedDevices` class to
+    support dehydrated devices.
+    ([#104](https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/104))
+
```

Comment:
please can we put more recent changelog entries at the top?

Reply:
Oh, I purposely put it at the bottom, because I mis-remembered the convention. :facepalm:

Reply:
yeahh... my thinking goes, if new *versions* go at the top, then by extension, so should individual entries, so that the newest stuff is always at the top.